client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -179,6 +194,31 @@ const convertToApiPackage = pckg => {
 		'weight',
 		'signature',
 	] );
+	if ( customsItems ) {
+		apiPckg.contents_type =
+			'other' !== pckg.contentsType ? pckg.contentsType || 'merchandise' : pckg.contentsExplanation;
+		apiPckg.restriction_type =
+			'other' !== pckg.restrictionType
+				? pckg.restrictionType || 'none'
+				: pckg.restrictionExplanation;
+		apiPckg.abandon_on_non_delivery = Boolean( pckg.abandonOnNonDelivery );
+		apiPckg.itn = pckg.itn || '';
+		const getProductValue = productId =>
+			isNil( customsItems[ productId ].value )
+				? getProductValueFromOrder( state, productId, orderId, siteId )
+				: customsItems[ productId ].value;
+
+		apiPckg.items = uniqBy( pckg.items, 'product_id' ).map( ( { product_id } ) => ( {
```

Comment:
This is similar to [this comment about product variations](https://github.com/Automattic/wp-calypso/pull/24974/files#r203568135) - should we filter these by product variation ID instead of the product ID?